pom-dist.xml

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>net.sf.jt400</groupId>
+  <artifactId>jt400</artifactId>
+  <packaging>pom</packaging>
+  <version>20.0.0-SNAPSHOT</version>
+  <name>JT400</name>
+  <description>The Open Source version of the IBM Toolbox for Java</description>
+  <url>http://github.com/IBM/JTOpen</url>
+
+  <licenses>
+    <license>
+      <name>IBM Public License Version 1.0</name>
+      <url>https://github.com/IBM/JTOpen/raw/main/license.html</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <java.compiler.version>8</java.compiler.version>
+    <maven.compiler.source>7</maven.compiler.source>
+    <maven.compiler.target>7</maven.compiler.target>
+    <maven.build.timestamp.format>yyyy-MM-dd HH:mm:ss</maven.build.timestamp.format>
+    <sc.version>Development Build (built with mvn)</sc.version>
+  </properties>
+  <developers>
+    <developer>
+      <name>Jesse Gorzinski</name>
+      <email>jgorzins@us.ibm.com</email>
+      <organization>IBM</organization>
+      <organizationUrl>http://www.ibm.com</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <url>https://sourceforge.net/p/jt400/svn/HEAD/tree/trunk/</url>
```

Comment:
Why is this still pointing to SF?

Reply:
good catch!

Reply:
Fixed at https://github.com/IBM/JTOpen/pull/11/commits/8dd6075243f0f195ce4c976b2eb9a7bda680e92a